armory/utils/metrics.py

```diff
@@ -523,6 +711,9 @@ def values(self):
     def mean(self):
         return sum(float(x) for x in self._values) / len(self._values)
 
+    def append_inputs(self, *args):
```

Comment:
Was this left in for debug purposes?

Reply:
this method is used to store up all of the `y`'s and `y_pred`'s so that maP can be computed across all samples at once after inference has finished. `self._inputs` is used [here](https://github.com/lcadalzo/armory/blob/759-mAP-metric/armory/utils/metrics.py#L729) to compute mAP